src/shared/utils/redis.util.ts

```diff
@@ -1,43 +1,50 @@
-import { Injectable } from "@nestjs/common";
+import { Injectable } from '@nestjs/common';
 // import * as redis from "redis";
 import * as appConfig from '../../shared/configs/configuration';
 const redis = require('redis');
 
 @Injectable()
 export class RedisUtil {
-
-    private redisClient;
-
-    constructor() {
-        const appParams = appConfig.default();
-        const redisURL = { url: `redis://${appParams.cacheManagement.redis.username}:${appParams.cacheManagement.redis.password}@${appParams.cacheManagement.redis.host}:${appParams.cacheManagement.redis.port}` };
-        this.redisClient = redis.createClient(redisURL);
-    }
-
-    public convertDateToString(date: Date) {
-        const timestamp = new Date();
-        timestamp.setSeconds(0, 0);
-        return timestamp.toISOString();
-
+  private redisClient;
+
+  constructor() {
+    const appParams = appConfig.default();
+    const redisURL = {
+      url: `redis://${appParams.cacheManagement.redis.username}:${appParams.cacheManagement.redis.password}@${appParams.cacheManagement.redis.host}:${appParams.cacheManagement.redis.port}`,
+    };
+    this.redisClient = redis.createClient(redisURL);
+  }
+
+  public convertDateToString(date: Date) {
+    const timestamp = new Date();
+    timestamp.setSeconds(0, 0);
+    return timestamp.toISOString();
+  }
+
+  public async connect() {
+    try {
+      if (!this.redisClient.isOpen) {
+        await this.redisClient.connect();
+      }
+      console.log('Auth complete...');
+    } catch (err) {
+      console.log(err);
     }
+  }
 
-    public async connect() {
-        try {
-            if (!this.redisClient.isOpen) {
-                await this.redisClient.connect();
-            }
-            console.log('Auth complete...');
+  public async setValue(key: string, data: any) {
+    await this.redisClient.set(key, JSON.stringify(data));
+  }
 
-        } catch (err) {
-            console.log(err);
-        }
-    }
+  public async getValue(key: string) {
+    return this.redisClient.get(key);
+  }
 
-    public async setValue(key: string, data: any) {
-        await this.redisClient.set(key, JSON.stringify(data));
-    }
+  public async getAllBullQueueName(): Promise<string[]> {
+    const BULL_REGEX = 'bull:*:id';
+    await this.connect();
+    const bullRedisKeys: string[] = await this.redisClient.keys(BULL_REGEX);
 
-    public async getValue(key: string) {
-        return this.redisClient.get(key);
-    }
-}
\ No newline at end of file
+    return bullRedisKeys.map((redisKey) => redisKey.split(':')[1]);
+  }
```

Comment:
Bull not support API to get all queues, so we need query redis to get all queues keys then mapping it.
By default, bull queue key in redis will look like `bull:*:id`
https://github.com/OptimalBits/bull/issues/334